bazaarvoice/emodb
# postman tests added for databus unclaimall endpoint

## What Are We Doing Here?

adding new postman tests for database/1/{subscription}/endpoint 
## How to Test and Verify

1. Check out this PR
2. Export newly added file with tests as a collection in postman
3. Run collection in postman

Verify that all tests are passed.

## Risk
no risk, it is new code without dependency to existing code
### Level 

`Low`

### Required Testing

`Manual`

### Risk Summary

Add one or a few complete sentences about the possible risks or concerns for
this change.

no risk is expected due to this change not being connected to existing code.
## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
